### PR TITLE
0.2/681 leaving function small 2

### PR DESCRIPTION
**Pull request by charlesstrange2**

### CODE REVIEW

File: config/arc.php [hunk 1]
```diff
@@ -9,7 +9,11 @@
     */
 
     'leaving_reasons' => [
-        'won the lottery',
-        'moved to the moon',
+        'positive circumstances',
```
Comment (katjam) [inline config/arc.php:12]: Oh I liked mine. :-1:

**Review by katjam — APPROVED** [review]
Beautiful. :boar:

File: config/arc.php (hunk 1 above)
Comment (charlesstrange2) [reply config/arc.php:12]: I liked them too, but public beta will involve people with unknown humour tolerance... :-(

**Comment by katjam** [comment]
Sure sure... I know.

**Pull request merged**